site/assets/js/application.js

```diff
@@ -119,7 +134,9 @@
 
   // Insert copy to clipboard button before .highlight
   var btnTitle = 'Copy to clipboard'
-  var btnHtml = '<div class="bd-clipboard"><button type="button" class="btn-clipboard">Copy</button></div>'
+  // var btnHtml = '<div class="bd-clipboard"><button type="button" class="btn-clipboard">Copy</button></div>'
+  var btnHtml = '<div class="bd-clipboard"><button type="button" class="btn-clipboard" ><svg class="bi" width="1em" height="1em" fill="currentColor"><use xlink:href="#clipboard"/></svg></button></div>'
```

Comment:
This needs some accessibility love: I'd go with `role="img" aria-label="Copy"` on the SVG, just in case tooltip doesn't work or something.

And feel free to drop the previous line instead of commenting, I thought it was some docs :D